downstream/modules/platform/ref-aap-containerized-dns-config.adoc

```diff
@@ -8,13 +8,13 @@
 
 .Using unbound DNS
 
-To configure unbound DNS refer to link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_networking_infrastructure_services/ssembly_setting-up-an-unbound-dns-server_networking-infrastructure-services#proc_configuring-unbound-as-a-caching-dns-server_assembly_setting-up-an-unbound-dns-server[Chapter 2. Setting up an unbound DNS server Red Hat Enterprise Linux 9].
+To configure unbound DNS, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_networking_infrastructure_services/assembly_setting-up-an-unbound-dns-server_networking-infrastructure-services[Chapter 2. Setting up an unbound DNS server - Red Hat Enterprise Linux 9].
 
 .Using BIND DNS
 
-To configure DNS using BIND refer to link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_networking_infrastructure_services/assembly_setting-up-and-configuring-a-bind-dns-server_networking-infrastructure-services[Chapter 1. Setting up and configuring a BIND DNS server Red Hat Enterprise Linux 9].
+To configure DNS using BIND, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_networking_infrastructure_services/assembly_setting-up-and-configuring-a-bind-dns-server_networking-infrastructure-services[Chapter 1. Setting up and configuring a BIND DNS server - Red Hat Enterprise Linux 9].
 
-Optional: To have the installer automatically pick up and apply your AAP subscription manifest license, then follow this guide up to and including step 11, Tp generate a manifest file which can be downloaded for the installer refer to link:https://docs.ansible.com/ansible-tower/latest/html/userguide/import_license.html#obtain-sub-manifest[4. Import a Subscription — Ansible Tower User Guide v3.8.6].
+Optional: To ensure the installation program automatically picks up and applies your {PlatformNameShort} subscription manifest license, generate a manifest file. You can then download this manifest file for the installation program to refer to. To generate the manifest file, see link:https://docs.ansible.com/ansible-tower/latest/html/userguide/import_license.html#obtain-sub-manifest[4.1. Obtaining a subscriptions manifest - Ansible Tower User Guide v3.8.6].
```

Comment:
You may have to update this link before the end of the year. The Controller User Guide contains a section of creating a Subscription Manifest.

Reply:
Ah good to know. I can make a note to update this again when the controller docs are live if that is suitable? 
Is there an ETA on the Controller User Guide? I know yourself and Ed are working away on it.

Reply:
I can make a note to update this link again when the Controller User Guide is published if that's suitable?

I can update the link for now with that updated link you provided, thank you :)

Reply:
Maybe end of November if we're lucky. However the published Getting Started guide still has links to the Controller User docs, and they all have to be updated once it's been moved to Customer Portal

Reply:
Good to know. Is there a Jira ticket or a spreadsheet to track updating references and that type of work for the Controller docs?

Reply:
Not yet. It's something I have planned. But as were only just over half way through. I was going to wait until later on. You could create one and link it to the Epic AAP-15342.

Reply:
No problem, I can do that so there's a place to track all this type of info. You can cross it off your list! :)